recipes/stingray/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "stingray" %}
+{% set version = "0.1rc1.post2" %}
+{% set sha256 = "f3d17782118427203cf2bbc7ab3cb6625fb1f031b11bd2ec9924418167bb9bf2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - astropy
```

Comment:
The answer to this question may be "yes", but are scipy, astropy, and matplotlib required for build? I know numpy probably is to ensure you get the right minimum version.

Reply:
@mwcraig probably not. They are surely required for any reasonable use of the library though. Is it sufficient to put them under `run`? If so, we will take them out of the build section

Reply:
Yeah, if you could try removing them from build that would be great. Limits the bandwidth for downloads for builds.